app/src/main/java/fr/free/nrw/commons/WelcomeActivity.java

```diff
@@ -50,10 +48,17 @@ public void onCreate(Bundle savedInstanceState) {
             findViewById(R.id.finishTutorialButton).setVisibility(View.VISIBLE);
         }
 
-        ButterKnife.bind(this);
+        binding = ActivityWelcomeBinding.inflate(getLayoutInflater());
+        final View view = binding.getRoot();
+        setContentView(view);
```

Comment:
These 3 lines need to move above the 2 "if" blocks, as the _Beta_ flavor will want to access a component in the layout.  As it stands, line 48 will fail to find the component and NullPointerException.  I would also replace the "findViewById" with a reference to the `finishTutorialButton` from the binding.